openvdb/python/CMakeLists.txt

```diff
@@ -226,7 +227,7 @@ install(TARGETS
   ${PYOPENVDB_INSTALL_DIRECTORY}
 )
 
-install(FILES ${PYTHON_PUBLIC_INCLUDE_NAMES} DESTINATION include/openvdb/python)
+install(FILES ${PYTHON_PUBLIC_INCLUDE_NAMES} DESTINATION ${OPENVDB_INCLUDE_DIR}/python)
```

Comment:
There's a bug here. `OPENVDB_INCLUDE_DIR` is defined in openvdb/CMakeLists.txt so is unset in this conetxt and results in CMake attempting to perform a absolutely path install to /python. I suggest for now that we just move `set(OPENVDB_INCLUDE_DIR "${CMAKE_INSTALL_INCLUDEDIR}/openvdb")` to the root CMake.

Reply:
Actually, I think this variable needs to be renamed. The root CMake allows you to build components of vdb without building the core library. One of the variables in the FindOpenVDB module which lets you search for a VDB installation is `OPENVDB_INCLUDE_DIR` which will conflict with its intended use here, especially if we move this to the root. Can we also rename this variable to `OPENVDB_INSTALL_INCLUDEDIR`?

Reply:
Ah, good catch.  I pushed a new commit that resolves that issue by moving `OPENVDB_INSTALL_INCLUDEDIR` to the top-level.  cheers.